site/dat/docs/Proxy2JMX.md

```diff
@@ -0,0 +1,22 @@
+# Proxy2JMX Converter
+
+It's possible to convert Selenium test into JMeter JMX file. For this purpose Taurus uses BlazeMeter recorder so you need valid token. Lets see example config:
```

Comment:
This sentence should probably have a link to BlazeMeter Recorder.

Also, a short explanation of what is happening inside the service may be useful.